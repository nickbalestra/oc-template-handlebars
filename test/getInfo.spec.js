const getInfo = require('../packages/oc-template-handlebars/getInfo');

describe('getInfo method', () => {
  describe('when invoking the method', () => {
    test('should return the correct info', () => {
      const info = getInfo();
      delete info.version;
      expect(info).toMatchSnapshot();
    });
  });
});
