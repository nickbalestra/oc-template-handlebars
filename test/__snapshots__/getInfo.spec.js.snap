// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getInfo method when invoking the method should return the correct info 1`] = `
Object {
  "externals": Array [
    Object {
      "global": "Handlebars",
      "url": "https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.6/handlebars.runtime.min.js",
    },
  ],
  "type": "oc-template-handlebars",
}
`;
